src/wags_tails/hemonc.py

```diff
@@ -0,0 +1,152 @@
+"""Provide source fetching for HemOnc."""
+import logging
+import os
+import zipfile
+from pathlib import Path
+from typing import NamedTuple, Tuple
+
+import requests
+
+from .base_source import DataSource, RemoteDataError
+from .utils.downloads import download_http
+from .utils.storage import get_latest_local_file
+from .utils.versioning import parse_file_version
+
+_logger = logging.getLogger(__name__)
+
+
+class HemOncPaths(NamedTuple):
+    """Container for HemOnc file paths.
+
+    Since HemOnc distributes data across multiple files, this is a simple way to pass
+    paths for each up to a data consumer.
+    """
+
+    concepts: Path
+    rels: Path
+    synonyms: Path
+
+
+class HemOncData(DataSource):
+    """Provide access to HemOnc data source."""
+
+    _src_name = "hemonc"
+    _filetype = "csv"
+
+    @staticmethod
+    def _get_latest_version() -> str:
+        """Retrieve latest version value
+
+        :return: latest release value
+        :raise RemoteDataError: if unable to parse version number from data file
+        """
+        data_url = "https://dataverse.harvard.edu/api/datasets/export?persistentId=doi:10.7910/DVN/9CY9C6&exporter=dataverse_json"
+        r = requests.get(data_url)
+        r.raise_for_status()
+        try:
+            date = r.json()["datasetVersion"]["createTime"].split("T")[0]
+        except (KeyError, IndexError):
+            raise RemoteDataError(
+                "Unable to parse latest HemOnc version number from release API"
+            )
+        return date
+
+    def _download_handler(self, dl_path: Path, file_paths: HemOncPaths) -> None:
+        """Extract HemOnc concepts, relations, and synonyms files from tmp zipfile, and save
+        to proper location in data directory.
+
+        Since we need to do some special logic to handle saving multiple files, this method
+        is curried with a ``file_paths`` argument before being passed to a downloader method.
+
+        :param dl_path: path to temp data file
+        :param file_paths: container for paths for each type of data file
+        """
+        paths_dict = file_paths._asdict()
+        with zipfile.ZipFile(dl_path, "r") as zip_ref:
+            for file in zip_ref.filelist:
+                for path_type, path in paths_dict.items():
+                    if path_type in file.filename:
+                        file.filename = path.name
+                        zip_ref.extract(file, path.parent)
+        os.remove(dl_path)
+
+    def _download_data(self, version: str, outfile_paths: HemOncPaths) -> None:
+        """Download data file to specified location.
+
+        :param version: version to acquire
+        :param outfile_paths: locations and filenames for final data files
+        """
+        api_key = os.environ.get("HARVARD_DATAVERSE_API_KEY")
+        if api_key is None:
```

Comment:
Since env vars are strings 

```suggestion
        if not api_key:
```